### PR TITLE
refactor(icons): invalid names

### LINKED ISSUE #259: Invalid icons names
**Describe the bug**
After fixing the naming for the latest new icons (#251), I noticed that most of them have invalid naming.

**Current behavior**
There are some icons that are named after the context where they are being used in the MC.

**Expected behavior**
Icons should be named according to what the icons itself represents, not the context.

---

If we decide to rename them, we should document properly which icons got renamed to what, to make it simple to migrate.

See [proposal naming down below](https://github.com/commercetools/ui-kit/issues/259#issuecomment-441624081)

### PR DESCRIPTION
**Pull request by emmenko**
Fixes #259 
Closes #260 

Renamed icons [listed here](https://github.com/commercetools/ui-kit/issues/259#issuecomment-441624081)

image

### CODE REVIEW

File: src/components/icons/svg/angle-thin-left.react.svg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
+    <title>arrow-left</title>
```
Comment (emmenko) [inline src/components/icons/svg/angle-thin-left.react.svg]: PS: the names here are still "wrong". Should we update them as well?

File: src/components/icons/svg/angle-thin-left.react.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <desc>Created with sketchtool.</desc>
+    <defs></defs>
+    <g id="Icons" stroke="none" stroke-width="1" fill-rule="evenodd">
+        <g id="MC-icon-set" transform="translate(-24.000000, -936.000000)" fill="#000000">
+            <g id="Directions" transform="translate(24.000000, 888.000000)">
+                <g id="Arrow-Left" transform="translate(0.000000, 48.000000)">
```
Comment (emmenko) [inline src/components/icons/svg/angle-thin-left.react.svg]: Same here with the `id`

File: src/components/icons/svg/angle-thin-left.react.svg (hunk 1 above)
Comment (tdeekens) [reply src/components/icons/svg/angle-thin-left.react.svg]: Uff. Good one. No opinion 😐

Comment (emmenko) [reply src/components/icons/svg/angle-thin-left.react.svg]: I went ahead and fixed the names: https://github.com/commercetools/ui-kit/pull/262/commits/f38e0124a19214bd2a286ea10214473406938ac6

I also noticed some inconsistency in the SVG structures. I already checked and notified the design team about it.

**Pull request merged**